code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -771,6 +771,8 @@
 	path = /mob/living/carbon/human/diona
 	default_language = "Galactic Common"
 	language = "Rootspeak"
+	speech_sounds = list('sound/voice/dionatalk1.ogg')
```

Comment:
If was taken from somewhere that required any attribution, we need to state it in a comment. Please link to the licence where the sound is from?